docs/conf.py

```diff
@@ -56,7 +56,7 @@
 # built documents.
 #
 # The short X.Y version.
-version = '1.0.0'
+version = pydocstyle.__version__
 # The full version, including alpha/beta/rc tags.
 release = '1.0.0'
```

Comment:
This should probably be equal to version

Reply:
good point, I usually do that. but since there it wasn't used in `rst` I did not change it.